Stackup-Rwanda/knights-bn-frontend
# #170947618 view new Notifications.

#### What does this PR do?
- allow a user to view his/her unread notifications.
#### Description of Task to be completed?
- allow a user to view his/her new notifications.
#### How should this be manually tested?
- run `git clone https://github.com/Stackup-Rwanda/knights-bn-frontend.git` to clone the repo.
- run `cd knights-bn-backend` in your terminal to enter into the repository.
- run `git fetch ` command to download commits, files, and refs from a remote repository into your local repo.
- run `git checkout ft-approved-notification-170947618` to switch to this branch.
- run `npm install` to install dependencies.
- run `npm run dev-start` to start the application.
  you will see the login page then login with your email or password 
```
        "email":"alain.maxime@gmail.com",
	"password":"Niyonkuru@1"
```
- use `http://localhost:8080/request` to create a request.

- log in again as an administrator.
```
administrator user:

        "email":"william.ishimwe@andela.com",
	"password":"Password@1"
```
- Go to pending requests then approve your request.
- pending request URL: - use `http://localhost:8080/pending` to create a request.

- login again into your account you will see the notification of the approved request.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)

      new notifications

<img width="1438" alt="Screen Shot 2020-06-03 at 10 22 34" src="https://user-images.githubusercontent.com/50768983/83613644-5fd0b180-a584-11ea-8894-08d012b0de8c.png">

    see notifications

<img width="1429" alt="Screen Shot 2020-06-03 at 10 22 42" src="https://user-images.githubusercontent.com/50768983/83613629-59423a00-a584-11ea-98bf-8b7438fa423d.png">
